mcs/tools/mkbundle/mkbundle.cs

```diff
@@ -351,11 +353,20 @@ static int Main (string [] args)
 			case "--bundled-header":
 				bundled_header = true;
 				break;
+			case "--aot-compile":
```

Comment:
It might be easier to merge the two arguments together, since --aot-args seems to be required, or --aot-args needs to be made optional.